source/common/matcher/BUILD

```diff
@@ -25,6 +25,14 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "validation_visitor_lib",
+    hdrs = ["validation_visitor.h"],
+    deps = [
+        "//include/envoy/matcher:matcher_interface",
+    ],
+)
+
```

Comment:
here - it needs to be a new line - a blank line is what is throwing the lint error

Reply:
the diff from fix format should fix it i guess

Reply:
ok let me give it a go, I guess I the fact that the output doesn't clearly differentiate between the two was confusing

Reply:
the pre formatting is to catch ~errors that are language-independent - so it doesnt catch things like extra lines

the format script lints according to langs and picks this up  -  which takes a lot longer 8/

Reply:
hmm - thats not quite correct - it more runs quick tests tnat at least atm dont have an autofix